wrappers/matlab/option.m

```diff
@@ -92,6 +92,7 @@
         transmitter_frequency           (88)
         vertical_binning                (89)
         receiver_sensitivity            (90)
-        count                           (91)
+        safety_camera_mode              (94)
```

Comment:
what about the values between 91 and 94?

Reply:
They were not exposed to matlab (auto_limit) and that's ok..